shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -783,6 +815,27 @@ protected void onFirstFrameRendered() {}
      * to provide its own {@code FlutterEngine} instance.
      */
     @Nullable
-    FlutterEngine getFlutterEngine(@NonNull Context context);
+    FlutterEngine provideFlutterEngine(@NonNull Context context);
+  }
+
+  /**
+   * Configures a {@link FlutterEngine} after it is created, e.g., adds plugins.
```

Comment:
Based on this description, it makes me assume that the default FlutterActivity would do something here but it just no-ops.

Reply:
Talked offline, resolving old comments.